src/ui/public/partials/paginated_selectable_list.html

```diff
@@ -0,0 +1,37 @@
+<form role="form" class="form-inline">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="input-group form-group finder-form col-md-9">
+        <span class="input-group-addon">
+          <i class="fa fa-search"></i>
+        </span>
+        <input
+          input-focus
+          ng-model="query"
+          placeholder="Filter..."
+          class="form-control"
+          name="query"
+          type="text"
+          autocomplete="off" />
+      </div>
+      <div class="finder-hit-count col-md-3">
+        <span>{{hitCount}} of {{hitCount}}</span>
+      </div>
+    </div>
+  </div>
+</form>
+<paginate list="hits" per-page="{{perPage}}">
+  <ul class="li-striped list-group list-group-menu">
+    <li
+      class="list-group-item list-group-menu-item"
+      ng-repeat="hit in page | filter: query | orderBy: hit.toString()"
+      ng-click="onSelect(hit, $event)">
+      <a ng-href="{{makeUrl(hit)}}" ng-click="preventClick($event)">
```

Comment:
This seems weird. It stands to reckon that there's only 2 mutually exclusive things you'd want to do:

1. Navigate to a new page. In which case you'd be following a link, so you'd want an href so as to keep open-in-tab functionality.
2. Call a function with the object

Why not make `userMakeUrl` and `userOnSelect` mutually exclusive and show a different element (`ng-if`) depending on which function is specified? That'll get rid of the need for the nested `ng-click` and the `preventDefault` stuff